scripts/release.sh

```diff
@@ -2,6 +2,9 @@
 
 set -e;
 
+rm -rf node_modules;
```

Comment:
I'm concerned this is going to slow down releases from GitHub Actions. CI always clones the repos and does a fresh `npm install` so we shouldn't need this. 

If this is needed for SPB can we add it there instead?